google-cloud-storage/src/test/java/com/google/cloud/storage/DefaultBufferedWritableByteChannelTest.java

```diff
@@ -56,10 +56,11 @@ void edgeCases() {
   @Property
   void bufferingEagerlyFlushesWhenFull(@ForAll("WriteOps") WriteOps writeOps) throws IOException {
     ByteBuffer buffer = ByteBuffer.allocate(writeOps.bufferSize);
+    AuditingBufferHandle handle = new AuditingBufferHandle(BufferHandle.handleOf(buffer));
```

Comment:
nit:could we just allocate the BufferHandle directly instead of doing a handleOf?

Reply:
Using the factory method rather than the constructor directly affords us an easy path to changing the actual implementing class if/when we need to.